deps/NNNN-rate-limiting-logic.md

```diff
@@ -0,0 +1,800 @@
+# Dynamo Rate Limit for Load Balancer Proposal
```

Comment:
What if we renamed this to 

Dynamo Request Back Pressure or Dynamo Request Rejection or Dynamo Dynamic Back Pressure?

Rate Limiting I think has some idea for limiting clients to a certain number of requests per second - and this gets confused with higher layers in the stack that would impose such limits.

The proposal here is not so much a rate limit as it is a back pressure / request rejection proposal - IIUC - is that fair?